inlong-manager/manager-web/src/main/resources/application-test.properties

```diff
@@ -89,6 +89,9 @@ sort.enable.zookeeper=false
 # cluster node timeout interval of heartbeat unit: second, the interval multiplied by 5 represents the true heartbeat timeout interval
 cluster.heartbeat.interval=6
 
+#
+reset.nodeStatus.enabled=false
```

Comment:
Ditto, otherwise, please remove the empty line.

Reply:
Fix.